src/checkout/ContactForm.tsx

```diff
@@ -0,0 +1,130 @@
+      <Grid.Column widthSM={Columns.Six}>
+        <FormSelectDropdown
+          id="country"
+          label="Country"
+          options={countries}
+          required
+        />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormSelectDropdown
+          id="usSubdivision"
+          label="State"
+          options={states}
+          required
+        />
+      </Grid.Column>
+    </Grid.Row>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="street1" label="Address 1" />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="street2" label="Address 2" />
+      </Grid.Column>
+    </Grid.Row>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="city" label="City" required />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Three}>
+        <FormInput id="postalCode" label="ZIP" required />
+      </Grid.Column>
+    </Grid.Row>
+  </Grid>
+)
+
+const CaContactForm: FC = () => (
+  <Grid>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormSelectDropdown
+          id="country"
+          label="Country"
+          options={countries}
+          required
+        />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="intlSubdivision" label="Province" />
+      </Grid.Column>
+    </Grid.Row>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="street1" label="Address 1" />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="street2" label="Address 2" />
+      </Grid.Column>
+    </Grid.Row>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="city" label="City" required />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Three}>
+        <FormInput id="postalCode" label="Postal Code" />
+      </Grid.Column>
+    </Grid.Row>
+  </Grid>
+)
+
+const IntlContactForm: FC = () => (
+  <Grid>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormSelectDropdown
+          id="country"
+          label="Country"
+          options={countries}
+          required
+        />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="intlSubdivision" label="State / Province / Region" />
+      </Grid.Column>
+    </Grid.Row>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="street1" label="Address 1" />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="street2" label="Address 2" />
+      </Grid.Column>
+    </Grid.Row>
+    <Grid.Row>
+      <Grid.Column widthSM={Columns.Six}>
+        <FormInput id="city" label="City / Town" required />
+      </Grid.Column>
+      <Grid.Column widthSM={Columns.Three}>
+        <FormInput id="postalCode" label="ZIP / Postal Code" />
+      </Grid.Column>
+    </Grid.Row>
+  </Grid>
+)
```

Comment:
Rather than having this grid pattern, we can make use of Flex box for a more flexible UI with less code

Reply:
Do you want me to refactor in the PR or should I create a new issue?